politeiawww/cmd/piwww/shortproposaldetails.go

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2017-2019 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/decred/politeia/politeiawww/cmd/shared"
+)
+
+// ShortProposalDetailsCmd retrieves the details of a proposal using its
+// token's prefix.
+type ShortProposalDetailsCmd struct {
```

Comment:
This should be abstracted away from the user, meaning it shouldn't be its own command.  The proposaldetails command should be able to accept either the short token or full token.

These should both work:

```
$ piwww proposaldetails a677e236cb2e0fdd485980cd5d789e668d00fdc5235d01e7345d2195b8679066
$ piwww proposaldetails a677e2
```